openslides/users/models.py

```diff
@@ -211,7 +210,6 @@ def save(self, *args, **kwargs):
         """
         if kwargs.get('update_fields') == ['last_login']:
             kwargs['skip_autoupdate'] = True
-            CollectionElement.from_instance(self)
```

Comment:
In general: The CollectionElement was not used to write the cache, since the ElementCache was merged into the master.

In this case: I don't see why it is necessary to update the cache. If only last_login was changed, then the cache does not have to be updated.